WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -227,6 +227,8 @@ def post_results
     end
     post = Post.new(title: title, body: body, author: current_user, world_readable: true)
     post_post(post)
+    comp.results_posted = Time.now
+    comp.save!
```

Comment:
OK, I'm never quite sure how that exactly works.

Still, I think it _should_ be last as a convention for readability and sanity.